### PR TITLE
Task 4361.dev Initial stylization "music_festivals"

### PR DESCRIPTION
**Pull request by Dexs1**
Before stylized blocks had to be rework them, and now they display the material correctly
https://redmine.devbranch.work/issues/4361

### CODE REVIEW

File: web/themes/custom/kitrab/src/scss/_musicfastival.scss [hunk 1]
```diff
@@ -0,0 +1,61 @@
+/*
+
+.field--name-field-image-news  {
+  img {max-width: 100%;
+  }
+}
+*/
+
```
Comment (devbranch-vitaliy) [inline web/themes/custom/kitrab/src/scss/_musicfastival.scss]: Dead code
```suggestion
```

File: web/themes/custom/kitrab/dist/style.css.map [hunk 2]
```diff
@@ -0,0 +1 @@
...lock-views {\n  text-align: center;\n  .field--name-title {\n    a:hover {\n      color: black;\n      font-weight: 600;\n      border-bottom: 3px solid #ac970c;\n    }\n  }\n}\n\nh2 > a {\n  color: black;\n  font-weight: 600;\n}\n\n.field--name-field-tagnews {\n  margin-top: -39px;\n  .field__item {\n\n    a {\n      color: black;\n      font-size: larger;\n      display: block;\n      background-color: white;\n      border: 1px solid black;\n      width: fit-content;\n      margin: 0 auto;\n      overflow: overlay;\n      padding-inline: 10px;\n    }\n  }\n}\n\n.field--name-field-slogan {\n  color: #767676;\n}\n\n.field--name-uid {\n    a{\n      color: #000;\n      &:hover{\n        color:#000;\n      }\n    }\n}\n.layout--threecol-section {\n  @include md {\n    display:grid;\n    grid-template-rows:1fr 1fr 1fr;\n\n  }\n}\n","@import \"breackpoints\";\n@import \"musicfastival\";\n@import \"global\";\n","$xs:200px;\n$sm:400px;\n$md:768px;\n$lg:992px;\n$xl:1200px;\n@mixin md (){\n  @media (max-width: $md) {\n    @content;\n  }\n}\n\n","$fonts-global:open-sans;\n\na, a.link\n{\n  border-bottom: none;\n    &:hover{\n    border-bottom: none;\n  }\n}\n\n"],"names":[],"sourceRoot":""}
```
Comment (devbranch-vitaliy) [inline web/themes/custom/kitrab/dist/style.css.map]: Add the file to the .gitignore

File: web/themes/custom/kitrab/dist/style.css [hunk 3]
```diff
@@ -1,4 +1,54 @@
 /*!************************************************************************************************************!*\
```
Comment (devbranch-vitaliy) [inline web/themes/custom/kitrab/dist/style.css]: Add the file to the .gitignore

**Review by devbranch-vitaliy — CHANGES_REQUESTED** [review]

**Comment by devbranch-vitaliy** [comment]
@Dexs1 Check my comments, please.

**Comment by Dexs1** [comment]
Removed gitignore files and was modified services.yml file

File: web/themes/custom/kitrab/src/scss/_breackpoints.scss [hunk 4]
```diff
@@ -0,0 +1,11 @@
+$xs:200px;
+$sm:400px;
+$md:768px;
+$lg:992px;
+$xl:1200px;
+@mixin md (){
+  @media (max-width: $md) {
+    @content;
+  }
+}
```
Comment (devbranch-vitaliy) [inline web/themes/custom/kitrab/src/scss/_breackpoints.scss]: We need to create mixins for some general cases.
```suggestion
@mixin media-breakpoint-max ($size){
  @media all and (max-width: $size) {
    @content;
  }
}
@mixin media-breakpoint-min ($size){
  @media all and (min-width: $size) {
    @content;
  }
}
```

File: web/themes/custom/kitrab/src/scss/_musicfastival.scss [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+.field--name-uid {
+    a{
+      color: #000;
+      &:hover{
+        color:#000;
+      }
+    }
+}
+.layout--threecol-section {
+  @include md {
```
Comment (devbranch-vitaliy) [inline web/themes/custom/kitrab/src/scss/_musicfastival.scss]: Implement a new mixin suggested above
```suggestion
  @include media-breakpoint-max($md) {
```

**Review by devbranch-vitaliy — CHANGES_REQUESTED** [review]

File: web/themes/custom/kitrab/src/scss/_musicfastival.scss [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+        color:#000;
+      }
+    }
+}
+.layout--threecol-section {
+  @include media-breakpoint-max($md) {
+  display:grid;
+  grid-template-rows:auto 1fr auto;
+
+      }
```
Comment (andriy-tkachuk) [inline web/themes/custom/kitrab/src/scss/_musicfastival.scss]:
```suggestion
  @include media-breakpoint-max($md) {
    display:grid;
    grid-template-rows:auto 1fr auto;
  }
```

File: web/themes/custom/kitrab/src/scss/_musicfastival.scss [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+  color: #767676;
+}
+
+.field--name-uid {
+    a{
+      color: #000;
+      &:hover{
+        color:#000;
+      }
+    }
```
Comment (andriy-tkachuk) [inline web/themes/custom/kitrab/src/scss/_musicfastival.scss]:
```suggestion
  a {
    color: #000;
    &:hover {
      color:#000;
    }
    }
```

File: web/themes/custom/kitrab/src/scss/_musicfastival.scss [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+
+h2 > a {
+  color: black;
+  font-weight: 600;
+}
+
+.field--name-field-tagnews {
+  margin-top: -39px;
+  .field__item {
+
```
Comment (andriy-tkachuk) [inline web/themes/custom/kitrab/src/scss/_musicfastival.scss]:
```suggestion
```

File: web/themes/custom/kitrab/src/scss/_musicfastival.scss [hunk 9]
```diff
@@ -0,0 +1,54 @@
+.block-views {
+  text-align: center;
+  .field--name-title {
+    a:hover {
+      color: black;
+      font-weight: 600;
+      border-bottom: 3px solid #ac970c;
```
Comment (andriy-tkachuk) [inline web/themes/custom/kitrab/src/scss/_musicfastival.scss:7]: Use variables for color

File: web/themes/custom/kitrab/src/scss/_global.scss [hunk 10]
```diff
@@ -0,0 +1,10 @@
+$fonts-global:open-sans;
+
+a, a.link
+{
```
Comment (andriy-tkachuk) [inline web/themes/custom/kitrab/src/scss/_global.scss]:
```suggestion
a, a.link {
```

**Review by andriy-tkachuk — CHANGES_REQUESTED** [review]

**Review by andriy-tkachuk — APPROVED** [review]

**Pull request merged**